ironfish-cli/src/commands/wallet/export.ts

```diff
@@ -45,7 +50,18 @@ export class ExportCommand extends IronfishCommand {
     const client = await this.sdk.connectRpc(local)
     const response = await client.exportAccount({ account })
 
```

Comment:
You will want to rebase on current `staging`, there will be conflicts from my `mnemonic` pr

Reply:
Rebased